app/controllers/users_controller.rb

```diff
@@ -0,0 +1,33 @@
+class UsersController < ApplicationController
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    user = User.create_invited_user(user_params)
+    if user.save
```

Comment:
This makes me wonder if it's worth having that decorator in the model? I mean - what's the point of moving all the controller logic into the model? I think I'd rather just make the decorator a private method in the controller. What are your thoughts?